scripts/changeset-version-with-docs.ts

```diff
@@ -6,6 +6,10 @@ import sh from 'shelljs';
   // Force exit on error
   sh.set(`-e`);
 
+  // Commit versions generated at pre-build step
+  sh.exec(`git add packages/versions/*`);
+  sh.exec(`git commit -m"ci(scripts): update versions"`);
+
```

Comment:
@camsjams The `versions` package now has a `prebuild` script that will update a `.ts` file, injecting them from the CI env vars, similar to how it is done for `versions.yml`.

I want to commit these changes from the CI, so the `FUELS` version is always up to date.

With this, we can use this package <sup>_(after #827 goes to `master`)_</sup> on the new doc engine.

Ref: https://github.com/FuelLabs/fuels-ts/pull/811#discussion_r1141633562

> cc @Torres-ssf